commit c0f332f1cba73940416477f0842e9892ba454362
Author: Konstantin Seurer <konstantin.seurer@gmail.com>
Date:   Tue May 6 22:12:58 2025 +0200

    vulkan/bvh: Add leaf.h to vk_bvh_includes
    
    Otherwise, the shader will not recompile when the file was modified.
    
    Reviewed-by: Natalie Vock <natalie.vock@gmx.de>
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/36965>

diff --git a/src/vulkan/runtime/bvh/meson.build b/src/vulkan/runtime/bvh/meson.build
index 32141d65532..02b2afb4163 100644
--- a/src/vulkan/runtime/bvh/meson.build
+++ b/src/vulkan/runtime/bvh/meson.build
@@ -45,6 +45,7 @@ bvh_shaders = [
 vk_bvh_include_dir = dir_source_root + '/src/vulkan/runtime/bvh'
 
 vk_bvh_includes = files(
+  'leaf.h',
   'vk_build_helpers.h',
   'vk_build_interface.h',
   'vk_bvh.h',
